### PR TITLE
Triggers for o6s

### PR DESCRIPTION
**Pull request by imynwulf**
Added some basic triggers for o6s's DPS mechanics.

### CODE REVIEW

File: ui/raidboss/data/triggers/o6s.js [hunk 1]
```diff
@@ -4,21 +4,25 @@
...
-      tts: function(data, matches) {
+      infoText: function(data,matches) {
+        if (data.me != matches[1])
+	return 'Last Kiss on ' +data.ShortName(matches[1]);
       },
+      tts: 'last kiss',
     },
   ]
-}]
+}
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o6s.js]: You're missing a ] here.  This should be }], or raidboss will give you an error when you load this file.

File: ui/raidboss/data/triggers/o6s.js [hunk 2]
```diff
@@ -4,21 +4,25 @@
...
-      id: 'O6S Thing',
-      regex: /:1EDD:Alte Roite starts using/,
-      infoText: function(data, matches) {
-      },
-      tts: function(data, matches) {
+      id: 'O6S Demonic Stone',
+      regex: /1B:........:(\y{Name}):....:....:0001:0000:0000:0000:/,
+      alarmText: function(data,matches) {
+        if (data.me == matches[1])
+	  return 'Demonic Stone on YOU';
```
Comment (quisquous) [inline ui/raidboss/data/triggers/o6s.js]: Can you use spaces instead of tabs throughout this file?

Comment (imynwulf) [reply ui/raidboss/data/triggers/o6s.js]: Sorry about that. Github and VIM were both using tabs. I switched github to two space indents and recommitted.

**Comment by imynwulf** [comment]
Everything should be fixed!

**Comment by quisquous** [comment]
Thanks for these! :D

**Pull request merged**